### DISCUSSION TITLE
Is there a way to set background colors for everything, without setting it each time for each prompt?

### DISCUSSION THREAD

**Discussion by jasursadikov** [romkatv/powerlevel10k#2633] [General]
Is there a way to set background colors for everything, without setting it each time for each prompt?
I am doing in my `.zshrc` following thing:
```bash
BACKGROUND_COLOR=16

typeset -g POWERLEVEL9K_OS_ICON_BACKGROUND=$BACKGROUND_COLOR
typeset -g POWERLEVEL9K_DIR_BACKGROUND=$BACKGROUND_COLOR
typeset -g POWERLEVEL9K_PROMPT_ELEMENT_BACKGROUND=$BACKGROUND_COLOR
...
```
Which is quite annoying, is there a way to optimize that?

**Comment by romkatv** [comment]
You can define `POWERLEVEL9K_BACKGROUND` and remove per-segment background parameters.

**Comment by romkatv** [comment]
Here's an example: https://github.com/romkatv/powerlevel10k/blob/master/config/p10k-classic.zsh
